### ISSUE TITLE
Remove Affix component

### ISSUE THREAD

**Issue by ang-zeyu** [MarkBind/markbind#1301]
The vue-components package contains an `Affix.vue` component, which seems to be undocumented nor used anywhere.

Was this a long-ago deprecated feature?

**Comment by ang-zeyu** [comment]
@damithc did you come across this before? `<affix>...</affix>`

**Comment by damithc** [comment]
> @damithc did you come across this before? `<affix>...</affix>`

It was never a user-facing feature of MarkBind

**Comment by ang-zeyu** [comment]
Hmm just found this https://github.com/MarkBind/vue-strap/pull/44, but if it was never intended to be there we could remove it now

**Comment by damithc** [comment]
Does it work as intended, even when there are other sticky elements in the page such as topnav, sitenav, and pagenav?

**Comment by ang-zeyu** [comment]
seems like it was being used for the docs before https://github.com/MarkBind/markbind/issues/266.

just tried it out as well, some of the implementation (classes particularly) required to make it work had already been removed, so it dosen't work.

> Does it work as intended, even when there are other sticky elements in the page such as topnav, sitenav, and pagenav?

We switched to alternate implementations for these as well, could remove it as such. There's also not much value in having `<affix>...</affix>` over `<div class="position-sticky">` I think, since you generally shouldn't have to worry about content responsiveness (`<affix>` hiding it on smaller screens versus `position-sticky`) when writing content
